ReactQt/runtime/src/reactscrollviewmanager.cpp

```diff
@@ -125,31 +125,26 @@ void UbuntuScrollViewManager::addChildItem(QQuickItem* scrollView, QQuickItem* c
 namespace {
 static const char* component_qml = R"COMPONENT(
 import QtQuick 2.4
-import Ubuntu.Components 1.2
+import QtQuick.Controls 1.4
 
 Flickable {
-  id: flikka
-  clip: true
-  contentWidth: contentItem.childrenRect.width
-  contentHeight: contentItem.childrenRect.height
-//   Scrollbar {
```

Comment:
@MaxRis, I agree. ScrollBar supported since Qt 5.7, so we can't add it in current implementation. Left for future.